src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -24,7 +24,7 @@
 	/**
```

Comment:
Please align the comments.

Reply:
There's nothing to align. This is the standard javadoc layout.